mne/viz/epochs.py

```diff
@@ -634,14 +636,14 @@ def _prepare_mne_browse_epochs(params, projs, n_channels, n_epochs, scalings,
         lines.append(lc)
 
     times = epochs.times
-    data = np.zeros((params['info']['nchan'], len(times) * len(epochs.events)))
+    data = np.zeros((params['info']['nchan'], len(times) * n_epochs))
 
     ylim = (25., 0.)  # Hardcoded 25 because butterfly has max 5 rows (5*5=25).
     # make shells for plotting traces
     offset = ylim[0] / n_channels
     offsets = np.arange(n_channels) * offset + (offset / 2.)
 
-    times = np.arange(len(data[0]))
+    times = np.arange(len(times) * len(epochs.events))
```

Comment:
No, ``times`` is the length of the whole data whereas ``n_epochs`` is the viewport size.